RecipeApi/Service/UserContentService.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using RecipeApi.IService;
+
+namespace RecipeApi.Service
+{
+   public class UserContentService: IUserContentService
```

Comment:
You're overriding standard mechanism by .net.
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.getuserasync?view=aspnetcore-8.0